zulipterminal/helper.py

```diff
@@ -676,4 +677,7 @@ def open_media(controller: Any, media_link: str, message_id: int) -> None:
         'darwin': 'open',
     }
     command = image_viewers[sys.platform]
-    subprocess.run([command, img_path])
+    callback = partial(subprocess.run, [command, img_path])
+    question = urwid.Text("Your requested media has been downloaded."
+                          " Do you want to view it?")
+    controller.show_confirmation_overlay(question, callback)
```

Comment:
I know we have the controller here, but it'd be cleaner if we kept the prompting separate - maybe add a method to core instead?